clients/src/test/java/org/apache/kafka/common/utils/MockTime.java

```diff
@@ -19,33 +19,53 @@
  */
 public class MockTime implements Time {
 
-    private long nanos = 0;
-    private long autoTickMs = 0;
+    private final long autoTickMs;
+
+    // Values from `nanoTime` and `currentTimeMillis` are not comparable, so we store them separately to catch bugs
```

Comment:
Ah I see, maybe
 s/"so we store them separately to catch bugs where this is incorrectly assumed to be true"/"so we store them separately to **allow for catching bugs** where this is incorrectly assumed to be true"/